packages/generate/src/commandutil.ts

```diff
@@ -15,6 +15,8 @@ export interface CommandOptions {
   updateIgnoreFile?: boolean;
   useHttpClient?: boolean;
   future?: boolean;
+  strictTypeNames?: boolean;
+  polymorphismAsDiscriminatedUnions?: boolean;
```

Comment:
Maybe something like
```suggestion
  future?: Partial<Record<keyof typeof future, boolean>>;
```
where we could pull the keys from a default future object